### PR TITLE
Updated eligibility criteria

### PR DESCRIPTION
**Pull request by idazucchi**
This updates the Dataset suitability SOP - see [ticket](https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/608) 

Let me know if I've missed anything!

@gabsie if you want to take a look

### CODE REVIEW

File: docs/SOPs/dataset_suitability_SOP.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ parent: SOPs
 # HCA Dataset Eligibility criteria 
 
 ## Purpose of this document
-As wranglers, our current method of tracking new, in-progress (wrangling) and complete datasets is by indicating their wrangling status, priority and release in Ingest. When new datasets are added, we need to assess whether they are eligible for HCA. There are specific eligibility criteria that we use to decide whether a dataset is eligible or not. It is important that this information is available so that time is not wasted on wrangling datasets that are not eligible for HCA and/or cannot be ingested in our system. The purpose of this document is to list the specific criteria we use to assess dataset eligibility and our general approach to implement these. Any feedback on this process is very welcome and should be added to this document; please make it clear that it is feedback.
+As wranglers we track new, in-progress (wrangling) and complete datasets by adding them as projects in ingest. You can [add a project](https://contribute.data.humancellatlas.org/projects/register/autofill) automatically with a doi or an accession, or you can register one manually.
```
Comment (Wkt8) [inline docs/SOPs/dataset_suitability_SOP.md:10]:
```suggestion
As wranglers we track new, in-progress (wrangling) and complete datasets by adding them as projects in ingest. You can [add a project](https://contribute.data.humancellatlas.org/projects/register/autofill) automatically with a doi or an accession, or you can register one manually.
Additionally, datasets are automatically added every month by our [script] (https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_acknowledgement_SOP.html#populate-ingest-from-nxnse-single-cell-database).
```

**Review by Wkt8 — CHANGES_REQUESTED** [review]
Requesting changes - please review.

File: docs/SOPs/dataset_suitability_SOP.md [hunk 2]
```diff
@@ -7,7 +7,13 @@ parent: SOPs
...
+
+When we register a new project we need to indicate its priority, eligibility, status and other key information in ingest (see the dataset [acknowledgment SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_acknowledgement_SOP.html#manual-curation-after-addition)). Having all this information helps us to select quickly projects to wrangle and keeps us from wasting time on projects that cannot be ingested and/or are not eligible for the HCA.
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_suitability_SOP.md:12]:
```suggestion
When we register a new project we need to indicate its priority, eligibility, status and other key information in ingest (see the dataset [acknowledgment SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_acknowledgement_SOP.html#manual-curation-after-addition)). Having all this information helps the wranglers to quickly select the projects to wrangle, rejecting datasets that are not eligible for the HCA.
```

File: docs/SOPs/dataset_suitability_SOP.md [hunk 3]
```diff
@@ -7,7 +7,13 @@ parent: SOPs
...
+
+The purpose of this document is to list the specific criteria we use to assess dataset eligibility and our general approach to implement these. Any feedback on this process is very welcome and should be added to this document; please make it clear that it is feedback. In the longer term we are aiming for a software focused solution to this task.
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_suitability_SOP.md:14]: We should include how to add feedback (e.g. mail the wrangler team at wrangler-team@data.humancellatlas.org)

File: docs/SOPs/dataset_suitability_SOP.md [hunk 4]
```diff
@@ -25,13 +31,12 @@ To be eligible for this category, datasets must:
...
 To be eligible for this category, datasets must:
 - Be referenced in a peer-reviewed journal or be available as pre-print
-- Contain at least some samples derived from healthy human donors. 
 - Be derived from a recognised single cell technology
 
 (1)  https://contribute.data.humancellatlas.org/
 (2)  https://www.humancellatlas.org/publications/
 
-This criteria means a huge number of datasets are eligible and there needs to be a prioritisation approch in order to understand order datasets should be wrangled. See [HCA dataset triage SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_triage_SOP.html)
+This criteria means a huge number of datasets are eligible, and we need rules to assign priority in order to understand in which order the datasets should be wrangled. See [HCA dataset triage SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_triage_SOP.html)
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_suitability_SOP.md:39]:
```suggestion
This criteria means a great number of datasets are eligible, and we need rules to assign priority. See [HCA dataset triage SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_triage_SOP.html) for more information.
```

File: docs/SOPs/dataset_suitability_SOP.md [hunk 5]
```diff
@@ -40,18 +45,19 @@ It is easiest/most productive to apply this to a list of datasets that have alre
 
 ## Assess data availability
 
-Check the type of consent: if the data is open, this is suitable. If the data is managed access, has living European donors, unavailable or cannot be found, the dataset can now be included if there are publically sharable contributor generated matrices. If there is no sharable data the dataset should be down prioritised or possibly considered ineligible.
+Check the type of consent: if the data is open, this is suitable. If the data is managed access, has living European donors and has no cell count matrices available or cannot be found the dataset is ineligible. The dataset can now be included if there are publically sharable contributor generated matrices. If there is no sharable data the dataset should be down prioritised or possibly considered ineligible.
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_suitability_SOP.md:48]:
```suggestion
Check the type of consent: 
- Open: **suitable**. 
- Managed/Controlled access: Needs to at least have **publicly accessible count matrices**. If there is no shareable data the dataset should be down prioritised or possibly considered ineligible.

Additionally, GDPR is in consideration; if the dataset contains samples from living European donors, no externally obtained data or metadata can be added. We consider externally obtained data any type of information that is not obtained from public sources (e.g. the manuscript or archives such as GEO).
```

re-vamped this section a bit

File: docs/SOPs/dataset_suitability_SOP.md [hunk 6]
```diff
@@ -40,18 +45,19 @@ It is easiest/most productive to apply this to a list of datasets that have alre
...
 
 Check that the sequence data is available (similar to the above). If the sequence data is fully available in fastq format, this is suitable. If the sequence data cannot be found or is not in fastq format, then the dataset can still be included if there are publically sharable contributor generated matrices. If there is no sharable data in the form of fastq files or a gene expression matrix, the dataset should be down prioritised or possibly considered ineligible.
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_suitability_SOP.md:50]:
```suggestion
Check that the sequence data is available (similar to the above). If the sequence data is fully available in fastq format, this is suitable. If the sequence data cannot be found or is not in fastq format, then the dataset can still be included if there are publicly sharable contributor generated matrices. If there is no shareable data in the form of fastq files or a gene expression matrix, the dataset should be down prioritised or possibly considered ineligible.
```

File: docs/SOPs/dataset_suitability_SOP.md [hunk 7]
```diff
@@ -40,18 +45,19 @@ It is easiest/most productive to apply this to a list of datasets that have alre
...
+_**Demultiplexing**: In the past if the experimental design involved multiple samples which were pooled before library preparation, the resulting fastq files had to be demultiplexed by the sample barcode before being uploaded to ingest. This was required by the pipelines team, but this is no longer necessary and we can ingest multiplexed data._
 
 ## Assess Technology
 
-Is the technology supported by our defined HCA metadata schema? If we can’t capture the information appropriately, then the dataset is blocked until the schema chnages can be made to support the technology, but does not make it ineligible. (see [HCA dataset triage SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_triage_SOP.html)).
+Is the technology supported by our defined HCA metadata schema? If we can’t capture the information appropriately, then the dataset is blocked until the schema changes can be made to support the technology, but this does not make it ineligible. (see [HCA dataset triage SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_triage_SOP.html)).
```
Comment (ESapenaVentura) [inline docs/SOPs/dataset_suitability_SOP.md:58]:
```suggestion
Is the technology supported by our defined HCA metadata schema? If we can’t capture the information appropriately, then the dataset is blocked until the schema changes can be made to support the technology, but this does not make it ineligible. (see [HCA dataset triage SOP](https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/dataset_triage_SOP.html)).

To check wether we have already discussed about a specific technology type, please refer to the [assay cheat sheet](https://docs.google.com/spreadsheets/d/1H9i1BK-VOXtMgGVv8LJZZZ9rbTG4XCQTBRxErdqMvWk/edit#gid=0)
```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]

File: docs/SOPs/dataset_suitability_SOP.md (hunk 3 above)
Comment (idazucchi) [reply docs/SOPs/dataset_suitability_SOP.md:15]: We should add the email to a user facing document, maybe the submission guidelines?

**Comment by idazucchi** [comment]
I've applied the suggestions and I'm merging to master

**Pull request merged**